ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitter.scala

```diff
@@ -104,6 +104,8 @@ private[kvutils] class TransactionCommitter(
             rejections.reject(
               DamlTransactionRejectionEntry.newBuilder
                 .setSubmitterInfo(transactionEntry.submitterInfo)
+                // the definite answer is false, as the rank-based deduplication is not yet implemented
```

Comment:
I think the below may grasp better the reason for returning a non-definite answer:
```suggestion
                // No duplicate rejection is a definite answer as the deduplication entry will eventually expire.
```